### DISCUSSION TITLE
可不可以考虑加入类似时间锁一样使用到了一定时间就弹出锁定程序的功能

### DISCUSSION THREAD

**Discussion by lxymahatma** [Planshit/Tai#23] [Ideas]
Tai可以监督你使用了多久的软件的话，可不可以加入设置一个时间段（或者使用时长），不在这个时间段（超出使用时长）之后就自动退出程序并锁定程序（或者打开之后就立刻关闭程序）

**Comment by noberumotto** [comment]
已经考虑在开发计划中了，但是需要更多的意见。我的想法是用户设定限额超出后由用户决定如何处理，比如执行CMD命令，都是可以自由控制的。但是不会倾向于家长监管的那种模式。

**Reply by lxymahatma** [reply]
其实主要是有些时候自己管不住自己😂本来计划好的每天每个时间段的事情然后有些时候一不小心哪里就看多了所以就破坏整个计划了（）这种情况下可能时间段会好点（当然我觉得普通超出限额也可以）毕竟工程量有点大

**Reply by noberumotto** [reply]
🤣毕竟这不是监管软件，就算限制住你启动你还是可以通过退出tai脱离限制，所以主要还是靠自觉。

**Reply by lxymahatma** [reply]
😂也是

**Reply by PinkD** [reply]
> 🤣毕竟这不是监管软件，就算限制住你启动你还是可以通过退出tai脱离限制，所以主要还是靠自觉。

个人感觉 Tai 还是 [KISS](https://en.wikipedia.org/wiki/KISS_principle) 比较好，正如 README 里写的，「👻在Windows上记录软件使用时长」

**Comment by rebron1900** [comment]
用处不大，电脑上多的是让你分心的东西。
这个得靠自己自律。

**Reply by lxymahatma** [reply]
😂有些时候游戏一打就是几个小时控制不了自己（）自控太差

**Reply by yongpz** [reply]
最后有一定概率会发现这就像闹钟一样，响了也会睡————锁定你也会想办法给他关掉

**Reply by Yinr** [reply]
打本的时候时间到了被强退，然后怒而删掉了 Tai，重新打开游戏😂
